samples/SocketsSample/wwwroot/hubs.html

```diff
@@ -78,6 +78,8 @@ <h1 id="head1"></h1>
 }
 
 let transportType = signalR.TransportType[getParameterByName('transport')] || signalR.TransportType.WebSockets;
+let hubRoute = getParameterByName('hubType') || "default";
+console.log("Hub Route: " + hubRoute);
```

Comment:
Feel free to file an issue - we probably also want uglify/minify/{whatever}-ify the version of the client we ship. This, however isn't TS - it is javascript embedded directly in html.